src/map/vana_time.cpp

```diff
@@ -202,13 +202,13 @@ TIMETYPE CVanaTime::SyncTime()
         lastTickedHour = m_vHour;
         switch (m_vHour)
         {
-            case  0: m_TimeType = TIME_NIGHT;   return TIME_MIDNIGHT;
-            case  4: m_TimeType = TIME_NEWDAY;  return TIME_NEWDAY;
-            case  6: m_TimeType = TIME_DAWN;    return TIME_DAWN;
-            case  7: m_TimeType = TIME_DAY;     return TIME_DAY;
-            case 17: m_TimeType = TIME_DUSK;    return TIME_DUSK;
-            case 18: m_TimeType = TIME_EVENING; return TIME_EVENING;
-            case 20: m_TimeType = TIME_NIGHT;   return TIME_NIGHT;
+            case  0: m_TimeType = TIME_MIDNIGHT; return TIME_MIDNIGHT;
```

Comment:
While I was investigating where to put `GetDayElement` (ie: "here in the time object?") I came across:
`case  0: m_TimeType = TIME_NIGHT`, and _assumed_ it should be `TIME_MIDNIGHT`. Other line diffs were just spacing since `MIDNIGHT` is longer than `EVENING`.

Yell at me if my assumption was wrong, @Kreidos

Reply:
Whoops! That one wasn't me, but I did miss it when I did the TOTD fix. Good catch.